### ISSUE TITLE
1.10.2 Can not access DankNull Dock GUI or set the extraction options

### ISSUE THREAD

**Issue by Sarelth** [p455w0rd/p455w0rd-s-Things#57]
Playing in the Project Ozone Lite modpack. Can not access the Danknull Dock GUI at all and in the Dank GUI there is no option to set the Extraction of items with the dock. If the Danknull is set on a dock for a few seconds it extracts the currently selected item though. 

Possibly might have something to do with the modpack configs, but currently can not find anything about it.

**Comment by p455w0rd** [comment]
nah..i just don't have time to update this mod any more tbh..it is really buggy at this point and I don't see any point in the near future where I will have time to dedicate to working on it as I've just started college..sorry

**Comment by Sarelth** [comment]
Thats perfectly okay, hope college goes well for you and I hope you can eventually get some time for the mod again, I love the concept and it makes mining so much less inventory cloggy.
